src/lib/y2network/sysconfig/hostname_writer.rb

```diff
@@ -74,7 +74,7 @@ def update_sysconfig_dhcp(hostname, old_hostname)
       # @param hostname [Y2Network::Hostname] Hostname configuration
       def update_hostname(hostname)
```

Comment:
What about adding a unit test for this writer?

Reply:
It seems that the `SENDMAIL` ... constant is not used, so probably copied from the `dns_writer`

https://github.com/yast/yast-network/blob/e7d6c742f4617ba4dd0cbd5cdbeba0ece4053195/src/lib/y2network/sysconfig/hostname_writer.rb#L45